clash/force_of_will.tex

```diff
@@ -79,10 +79,10 @@ \subsection*{\MakeUppercase{Additional Rules}}
   \item A player may not use the Diplomacy Card to skip Combat on the Dragon Utopia Field.
   \item When a Hero moves to the Dragon Utopia Field, they must fight, depending on chosen difficulty, the following number of Neutral Units:
   \begin{itemize}
-    \item \textbf{Easy:} 1 × \svgunit{azure}, 2 × \svgunit{silver}, 1 × \svgunit{bronze}
-    \item \textbf{Normal:} 1 × \svgunit{azure}, 1 × \svgunit{golden}, 2 × \svgunit{silver}
-    \item \textbf{Hard:} 1 × \svgunit{azure}, 3 × \svgunit{golden}
-    \item \textbf{Impossible:}  2 × \svgunit{azure}, 2 × \svgunit{golden}
+    \item \textbf{Easy:} 1 × \azure\, 2 × \silver\, 1 × \bronze\
+    \item \textbf{Normal:} 1 × \azure\, 1 × \golden\, 2 × \silver\
+    \item \textbf{Hard:} 1 × \azure\, 3 × \golden\
```

Comment:
why all the \ at the end?

Reply:
That's a weird LaTeX thing. If a command doesn't use any args, all the whitespace after it is ignored. It doesn't make a difference here, but it does for something like: `A Few \bronze\ Units`, in which case the SVG would be glued to the word Units.

Reply:
I just replaced everything with sed.